R/kenwardroger.R

```diff
@@ -0,0 +1,198 @@
+#' Obtain Kenward-Roger Adjustment Components
+#'
```

Comment:
Please add a one sentence description as a minimum for all functions. E.g. here:

```suggestion
#'
#' Obtains the components needed downstream for the computation of Kenward-Roger degrees of freedom.
#'
```